app-authz-spring-security/README.md

```diff
@@ -0,0 +1,97 @@
+app-authz-spring-security: Spring Security Application using Authorization Services
+===================================================
+
+Level: Beginner
+Technologies: SpringBoot, Spring MVC, Spring Security
+Summary: SpringBoot, Spring MVC, Spring Security
+Target Product: <span>Keycloak</span>
+Source: <https://github.com/keycloak/keycloak-quickstarts>
+
+
+What is it?
+-----------
+
+The `app-authz-spring-security` quickstart demonstrates how to write a Spring Security application where both authentication and
+authorization aspects are managed by <span>Keycloak</span>.
+
+This application tries to focus on the authorization features provided by <span>Keycloak</span> Authorization Services, where resources are
+protected by a set of permissions and policies defined in Keycloak itself and access to these resources are enforced by a policy enforcer
+that intercepts every single request to the application.
+
+In this application, there are three paths protected by specific permissions in <span>Keycloak</span>:
+
+* **/protected**, where access to this page is based on the evaluation of permissions associated with a resource **Protected Resource** in <span>Keycloak</span>. Basically,
+any user with a role *user* is allowed to access this page.
+
+* **/protected/premium**, where access to this page is based on the evaluation of permissions associated with a resource **Premium Resource** in <span>Keycloak</span>. Basically,
+only users with a role *user-premium* is allowed to access this page.
+
+* **/protected/alice**, where access to this page is based on the evaluation of permissions associated with a resource **Alice Resource** in <span>Keycloak</span>. Basically,
+only user *alice* is allowed to access this page.
+
+The home page (home.ftl) also demonstrates how to use a ``AuthorizationContext`` instance to check for user`s permissions and hide/show
```

Comment:
There is a back tick in "user`s" that should instead be an apostrophe, "user's".